src/server/services/pipeline-in-progress/validateReadings.js

```diff
@@ -32,7 +32,7 @@ function validateReadings(arrayToValidate, conditionSet, meterName = undefined)
  * @param {Moment} maxDate inclusive latest acceptable date (won't be rejected)
  * @param {number} maxError maximum number of errors to be reported, ignore the rest
```

Comment:
How would I add something to the JSdoc?

Reply:
You can put
`` * @param {string} meterName The name of the meter``
at the end of the param list. It will show up as part of the JSDoc in VSC (or if manually created which we don't do).